friends/friendModel.js

```diff
@@ -0,0 +1,39 @@
+//here we'll create a shape for all our documents that come in. We will build a model for bears. 
+const mongoose = require('mongoose');
+
+//The model is where we define the schema. It's a blueprint of what our collection will look like
+// collection will be what our model is called
+
+// Schema
+// {
+//     firstName: "Jane", // String, required
+//     lastName: "Doe",  // String, required
+//     age: 18, // Number, required, should be an integer between 1 and 120
+//     createdOn: Mon Aug 14 2017 12:50:16 GMT-0700 (PDT) // Date, required, defaults to current date
+//   }
+
+const FriendSchema = new mongoose.Schema({
+    firstName: {
+        type: String,
+        required: true
+    }, 
+    lastName: {
+        type: String,
+        required: true
+    },
+    age: {
```

Comment:
I know that the README asks you to validate the age (between 1-120) in the endpoint, but you could also do it by adding more properties to your Mongoose Schema using min and max validators.